api/package.json

```diff
@@ -24,9 +24,9 @@
     "@medic/server-checks": "^1.0.1",
     "@medic/task-utils": "^1.0.0",
     "@medic/tombstone-utils": "^1.0.0",
-    "medic-transitions-beta": "^0.0.5",
     "@medic/translation-utils": "^1.0.1",
     "@medic/view-map-utils": "^1.0.0",
+    "@scdf/transitions": "^1.1.0",
```

Comment:
We do this so we can publish the packages before doing the actual merge.

Once this is ready to roll I'll go through and create a proper PR that uses the medic org etc